src/cls/cmpxattr/ops.h

```diff
@@ -0,0 +1,50 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2024 IBM Corp.
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ */
+
+#pragma once
+
+#include "types.h"
+#include "include/encoding.h"
+#include <time.h>
+#include "include/utime.h"
+#include "common/ceph_time.h"
+
+namespace cls::cmpxattr {
+  struct cmp_vals_set_vals_op {
```

Comment:
nit: google style guide says not to indent namespaces

https://google.github.io/styleguide/cppguide.html#Namespace_Formatting